cli/src/test_main.rs

```diff
@@ -0,0 +1,34 @@
+use std::time::{Instant};
```

Comment:
omit this file entirely, we will write real [benchmarks](https://doc.rust-lang.org/unstable-book/library-features/test.html) and [tests](https://doc.rust-lang.org/book/ch11-01-writing-tests.html) for the lexer (and everything else) instead